### DISCUSSION TITLE
Will the raw data of WikiKG90M-LSC  be released?

### DISCUSSION THREAD

**Discussion by jianyucai** [snap-stanford/ogb#136] [WikiKG90M-LSC]
Hi, I noticed that the `raw` directory of `wikikg90m_kddcup2021` is empty. Will the raw data be released? 

Thanks.

**Comment by weihua916** [comment]
Yes, we will eventually release the raw data, but we have not yet decided the timing for that.

**Reply by jianyucai** [reply]
Thank you for your kind response.

**Reply by hhr114** [reply]
> Yes, we will eventually release the raw data, but we have not yet decided the timing for that.

Hi, Weihua,
I notice that the KDD Cup 2021 is over. Would u plz tell me when will the raw data be released?

Thanks : )

**Comment by zhengfengL** [comment]
Hi, I notice that the KDD Cup 2021 is over, when will the raw data be released?

Thanks : )

**Reply by weihua916** [reply]
We are planning to continue hiding the test labels. We will open the evaluation server in the next few months so that the community can continue to work on the LSC datasets.

For WikiKG90M, we are most likely going to deprecate it and create a new version of it that does not provide any candidate test tails. This is because the accuracy is already saturated during the KDD cup (0.97 MRR), and we think it is useful to provide a greater and more realistic challenge to the community. We will keep the community updated.

**Reply by hhr114** [reply]
> We are planning to continue hiding the test labels. We will open the evaluation server in the next few months so that the community can continue to work on the LSC datasets.
> 
> For WikiKG90M, we are most likely going to deprecate it and create a new version of it that does not provide any candidate test tails. This is because the accuracy is already saturated during the KDD cup (0.97 MRR), and we think it is useful to provide a greater and more realistic challenge to the community. We will keep the community updated.

Could you plz release the coresponding meaning of each entity/relation id in present training dataset? We need to visualize some part of the knowledge graph and do case studies for paper writing. By providing entity/relation dicts of training data, I think WikiKG90M will continue to do good for the community. Thanks.

**Reply by weihua916** [reply]
Whether we can release the name is still TBD. It’s likely we cannot for the time being since we want to avoid test data leakage.

**Reply by cj2001** [reply]
I would like to +1 the request to have the labeled data (i.e. the conversion between integers and node/relationship labels) provided.  This is a tremendously valuable data set that may in the community can benefit from.  Even if it is the deprecated set, it would still be of great benefit to all.

**Reply by sanxing-chen** [reply]
> Whether we can release the name is still TBD. It’s likely we cannot for the time being since we want to avoid test data leakage.

I am curious about why releasing the names of entities would cause test data leakage. @weihua916 could you elaborate on this? Hiding the names prevents the development of inductive models with built-in text encoders.

**Reply by weihua916** [reply]
It's because you can use the names to search over the WikiData (e.g., Japan: https://www.wikidata.org/wiki/Q17). For now, we have already encoded text for you. You can use the provided embedding features.

**Reply by sanxing-chen** [reply]
Thanks for the reply, fair enough.

**Comment by jannatmeem95** [comment]
Hi! Will the raw data be released? Or is there any publicly available small chunk/snippet of the raw dataset?

**Comment by weihua916** [comment]
Hi everyone, we just released the raw text data: https://groups.google.com/g/open-graph-benchmark/c/R0SKtj9qQyE
